src/Numerics/ODESolvers/StrongStabilityPreservingRungeKuttaMethod.jl

```diff
@@ -149,6 +149,103 @@ end
+time stepping object is intended to be passed to the `solve!` command.
+
+This uses the second-order, 2-stage, strong-stability-preserving, Runge--Kutta scheme
+of Shu and Osher (1988) (also known as Heun's method.)
+Exact choice of coefficients from wikipedia page for Heun's method :)
+
+### References
+    @article{shu1988efficient,
+      title={Efficient implementation of essentially non-oscillatory shock-capturing schemes},
+      author={Shu, Chi-Wang and Osher, Stanley},
+      journal={Journal of computational physics},
+      volume={77},
+      number={2},
+      pages={439--471},
+      year={1988},
+      publisher={Elsevier}
+    }
+    @article {Heun1900,
+       title = {Neue Methoden zur approximativen Integration der
+       Differentialgleichungen einer unabh\"{a}ngigen Ver\"{a}nderlichen}
+       author = {Heun, Karl},
+       journal = {Z. Math. Phys},
+       volume = {45},
+       pages = {23--38},
+       year = {1900}
+    }
+"""
+function SSPRK22Heuns(F, Q::AT; dt = 0, t0 = 0) where {AT <: AbstractArray}
+    T = eltype(Q)
+    RT = real(T)
+    RKA = [RT(1) RT(0); RT(1//2) RT(1//2)]
+    RKB = [RT(1), RT(1 // 2)]
+    RKC = [RT(0), RT(1)]
+    StrongStabilityPreservingRungeKutta(F, RKA, RKB, RKC, Q; dt = dt, t0 = t0)
+end
+
+"""
+    SSPRK22Ralstons(f, Q; dt, t0 = 0)
+
+This function returns a [`StrongStabilityPreservingRungeKutta`](@ref) time stepping object
+for explicitly time stepping the differential
+equation given by the right-hand-side function `f` with the state `Q`, i.e.,
+
+```math
+  \\dot{Q} = f(Q, t)
+```
+
+with the required time step size `dt` and optional initial time `t0`.  This
+time stepping object is intended to be passed to the `solve!` command.
+
+This uses the second-order, 2-stage, strong-stability-preserving, Runge--Kutta scheme
+of Shu and Osher (1988) (also known as Ralstons's method.)
+Exact choice of coefficients from wikipedia page for Heun's method :)
+
+### References
+    @article{shu1988efficient,
+      title={Efficient implementation of essentially non-oscillatory shock-capturing schemes},
+      author={Shu, Chi-Wang and Osher, Stanley},
+      journal={Journal of computational physics},
+      volume={77},
+      number={2},
+      pages={439--471},
+      year={1988},
+      publisher={Elsevier}
+    }
+    @article{ralston1962runge,
+      title={Runge-Kutta methods with minimum error bounds},
+      author={Ralston, Anthony},
+      journal={Mathematics of computation},
+      volume={16},
+      number={80},
+      pages={431--437},
+      year={1962},
+      doi={10.1090/S0025-5718-1962-0150954-0}
+    }
+"""
+function SSPRK22Ralstons(F, Q::AT; dt = 0, t0 = 0) where {AT <: AbstractArray}
+    T = eltype(Q)
+    RT = real(T)
+    RKA = [RT(1) RT(0); RT(10 // 12) RT(2 // 12)]
+    RKB = [RT(2 // 3), RT(3 // 4)]
+    RKC = [RT(0), RT(2 // 3)]
+    StrongStabilityPreservingRungeKutta(F, RKA, RKB, RKC, Q; dt = dt, t0 = t0)
```

Comment:
Is this correct? This is supposed to be Ralston's 2nd order method? Isn't this the correct Butcher table:
```
0    | 0      0
2/3  | 2/3    0
----------------
     | 1/4   3/4
```
I looked that the paper you posted in the docstring but it's not obvious to me where you're getting those numbers. Maybe I'm thinking of a completely different method.

Reply:
The Tableau's in the code are not in standard form, so we have to mangle them into the assumed form.

Someone needs to sit down and mangle this one correctly.

Reply:
That said, I think that what is in the code is not correct.

Reply:
@jkozdon can you document the conversion from the standard Butcher Tableau to the form used in Clima? Does this generalize the same for both the SSPRK methods and the LSRK methods?

Also, I think the Ralston coefficients in the code are wrong because Heun's method passed and Ralston's failed, which implies the numerics are bad.

Reply:
I just looked at what the scheme was doing in ['dostep!`](https://github.com/CliMA/ClimateMachine.jl/blob/12ca4ef3e45e9a2c185485e2254569f5f890abff/src/Numerics/ODESolvers/StrongStabilityPreservingRungeKuttaMethod.jl#L68-L126) and tried backed out what they need to be.

Obviously I made a mistake when I tried to do this.

I didn't implement these schemes, so I don't know what the rational behind the non-standard for is or what the conversion in general is.

Reply:
So a two stage method will be:
```
Q1 = Q
R1 = f(Q1)
Q2 = A[1,1] * Q + A[1,2] * Q1 + B[1] * dt * R1
R2 = f(Q2)
Q3 = A[2,1] * Q + A[2,2] * Q2 + B[2] * dt * R2 
Q = Q3
```
To get Ralston's method you need:
```
Q2 = Q + (2/3) * dt * f(Q)
```
which implies that
```
A[1,1] = 1
A[1,2] = 0
B[1] = 2/3
```
Then you need
```
Q3 = Q + (1/4) * dt f(Q) + (3/4) * dt * f(Q2)
   = A[2,1] * Q + A[2,2] * Q2 + B[2] * dt * R2 
   = A[2,1] * Q + A[2,2] * (Q + (2/3) * dt * f(Q)) + B[2] * dt * R2
   = (A[2,1] + A[2,2]) * Q + (2/3) * A[2,2] * dt * f(Q) + B[2] * dt * f(Q2)
```
which impies that
```
A[2,2] = 3 / 8
A[2,1] = 5 / 8
B[2] = 3/4
```

Reply:
> Does this generalize the same for both the SSPRK methods and the LSRK methods?

the LSRK tableaus  are in standard form.